webextensions/api/declarativeNetRequest.json

```diff
@@ -0,0 +1,1656 @@
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "excludedDomains": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "84",
+                  "notes": "Deprecated in Chrome 101, use <code>excludedInitiatorDomains</code> instead."
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "excludedInitiatorDomains": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "101"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "excludedRequestDomains": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "101"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "excludedRequestMethods": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "91"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "requestDomains": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "101"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
```

Comment:
`requestDomains` is not supported by shipping Safari. It is supported in Safari Technology Preview.